great_expectations/experimental/datasources/sql_datasource.py

```diff
@@ -331,7 +343,7 @@ def get_batch_list_from_batch_request(
 
 
 class SQLDatasource(Datasource):
-    """Adds a generic SQL datasource to the data context.
+    """A generic SQL datasource.
```

Comment:
This becomes the docstring for our `add_sql_datasource` method hence the imperative style. It used to say something similar to what you wrote until I got feedback the other way.

Reply:
Nice catch! Thanks! ✅